sdk/core/System.ClientModel/src/Internal/Logging/ClientModelLogMessages.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Net.NetworkInformation;
+using System.Text;
+using Microsoft.Extensions.Logging;
+
+namespace System.ClientModel.Internal;
+
+// The methods in this class should only ever be called from LoggingHandler
+internal partial class ClientModelLogMessages
+{
+    private const int RequestEvent = 1;
```

Comment:
If the Ids are intended to be common, should we split them out into a separate file referenced by both?

Reply:
Yeah this is a really good idea I will do that.